src/debug/ee/controller.cpp

```diff
@@ -9007,7 +9007,7 @@ bool DebuggerContinuableExceptionBreakpoint::SendEvent(Thread *thread, bool fIpC
     if (hitDataBp)
     {
         CONTEXT contextToAdjust;
-        bool adjustedContext = false;
+        BOOL adjustedContext = false;
```

Comment:
A nit - now that you've changed `bool` to `BOOL` can you please use `FALSE` instead of `false` here?